api/src/wfl/api/workloads.clj

```diff
@@ -137,6 +143,15 @@
              :pipeline pipeline
              :type     ::invalid-pipeline})))
 
+(defmethod retry
+  :default
+  [{:keys [pipeline] :as workload} _]
+  (throw
+   (ex-info "Failed to retry workflows - no such pipeline"
+            {:workload workload
+             :pipeline pipeline
+             :type     ::invalid-pipeline})))
```

Comment:
Maybe `_workflows` instead of just `_`?

Reply:
I don't think it's needed - the multimethod defines this aleady.

Reply:
I guess I care more about names than about types -- at least in Clojure.

Reply:
I don't see why it matters. It's ignored by the implementation and since "clojure doesn't care about types" what does it matter what anyone passes in?

Reply:
Because we want human beings to understand it.

Reply:
Again, I don't understand how that makes it more understandable. It's an unused parameter - it could be anything.

Reply:
The methods use it.